DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: janitor
 Title: Simple Tools for Examining and Cleaning Dirty Data
-Version: 0.2.1.9000
+Version: 0.3
```

Comment:
I think that convention is to bump the version now so that we can tell the difference between which of our code is using the prod version and which is using this version. I agree that a PR is not the way to do this -- you should probably commit directly. People will know it's a dev version because it's installed via install_github instead of cran. Alternatively, you could create a "v0.3" branch that I can submit PRs to and then we can pull that branch into master when you are satisfied. 

Can you do one of these soon as it will impact future PRs?